oauth2/fosite_store_sql.go

```diff
@@ -84,6 +91,10 @@ var Migrations = map[string]*dbal.PackrMigrationSource{
 	}, true),
 }
 
+type transactionKey struct{}
+
+var txKey = transactionKey{}
```

Comment:
This is fine! However, a more idiomatic would be by using int (optionally with iota)

```
type transactionKey int
const txKey transactionKey = iota
```